README.md

```diff
@@ -28,7 +28,7 @@ Presuming you are running some Linux with a `sudo` capable user, the following s
 
   ```bash
   git clone https://github.com/dune-community/Dockerfiles.git docker-for-daily-dune
-  ln -s docker-for-daily-dune/docker_* .
+  ln -s docker-for-daily-dune/docker_run.sh .
```

Comment:
Yes the mac had problems with it. 

Ah ok, I didn't know there was another file, we need. 
Apparently, I yet ignored it completely.